### DISCUSSION TITLE
What hardware recommendations?

### DISCUSSION THREAD

**Discussion by herewithme** [healthchecks/healthchecks#424] [General]
Would it be very interesting to add the recommendations in terms of hardware to your wiki?

The minimum number of CPU / RAM for the project?
For example, for 100 checks at 5 minutes on average, what is the expected sizing?

Thanks in advance :)

**Comment by cuu508** [comment]
I agree! But, to be honest, I don't know what numbers to recommend. Let's see if we can collect the specs in this issue. Self-hosting people can perhaps chime in.

To get the conversation started:

**CPU** must be chosen proportionally to the traffic you expect during spikes. In your example, if all 100 checks receive a ping at the same instant, the CPU must be able to handle 100 requests (each with a new TLS handshake) per second. 

Could a 1 vCPU from a popular VPS provider handle it? I don't know, I  haven't benchmarked it.

**RAM** – Python, Django apps like memory. Things would be tight on 512MB RAM, but would probably be fine on 1GB. Again – needs to be tested.

**Comment by herewithme** [comment]
Haha, trick question if I understand correctly :)

**Comment by SuperSandro2000** [comment]
I don't hit the 100 requests an hour but mine is currently running with less than 200MB. I think 1GB should be more than enough for your use case.
